cumulusci/core/config/project_config.py

```diff
@@ -673,8 +673,9 @@ def _add_tasks_directory_to_python_path(self):
             self.logger.debug(f"Adding {directory} to tasks.__path__")
             tasks.__path__.append(directory)
         if get_debug_mode():
+            spec = getattr(self.source, "spec", ".")
             self.logger.debug(
-                f"After importing {self.source.spec}:  tasks.__path__ {tasks.__path__}"
+                f"After importing {spec}:  tasks.__path__ {tasks.__path__}"
```

Comment:
Logging change necessitated by the fact that NullSource (the root project's source) does not have a ".spec".